resume.go

```diff
@@ -7,15 +7,15 @@ import (
 
 // Export extracts dtls state and inner connection from an already handshaked dtls conn
 func (c *Conn) Export() (*State, net.Conn, error) {
```

Comment:
Exporting inner `net.Conn` is useful for testing but should be unnecessary for the real use.
Replacing `Export` by `ConnectionState` maybe better.

Reply:
Yea, I agree. I propose I'll change that in #237 too

Reply:
`crypto/tls` doesn't have a `Export` method but it is a requested feature. See https://github.com/golang/go/issues/25351#issuecomment-415357066.

Both @Sean-Der and me agreed that it was an interesting feature to add to `pion/dtls` and it was added here: https://github.com/pion/dtls/pull/41

I don't mind renaming/refactoring `Export` by `ConnectionState` even if this is a breaking change but just saying that the feature of exporting and resuming is there for a reason and shouln't be taken out, at least without discussing it.

Reply:
I didn't know about that!
The main difference between the `Export` and `ConnectionState` is that `Export` also exposes `net.Conn`. I'm not super familiar with the DTLS resume;  is the `ConnectionState` not sufficient for resuming (i.e. is `net.Conn` also required to resume a connection)?

Reply:
Answered here: https://github.com/pion/dtls/pull/237#issuecomment-605506634